src/util/certs/get-wildcard-cns-for-alias.ts

```diff
@@ -1,15 +1,21 @@
 import psl from 'psl';
 import { InvalidDomain } from '../errors-ts';
+import isWildcardAlias from '../alias/is-wildcard-alias';
+import extractDomain from '../alias/extract-domain';
 
-export default function getWildcardCNSForDomain(rawDomain: string) {
-  const parsedDomain = psl.parse(rawDomain);
+export default function getWildcardCNSForAlias(alias: string) {
```

Comment:
I like the camel casing like in the file above better.

```suggestion
export default function getWildcardCnsForAlias(alias: string) {
```